setup.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+from typing import List
+
+# setuptools doesn't support type hints for now:
+# https://github.com/pypa/setuptools/issues/2345
+# so we ignoring mypy checks on this package
+from setuptools import find_packages, setup
+
+
+def list_requirements(path: str) -> List[str]:
```

Comment:
I would rather move these files content in here and used `.[influxdb,kafka,elasticsearch]` somewhere in tox